rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1211,6 +1211,36 @@ void save_saveLastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage
 
     }
 
+    @Test
+    void save_saveFederationElection_shouldSaveInStorage() {
+
+        // Arrange
+
+        AddressBasedAuthorizer authorizer = getTestingAddressBasedAuthorizer();
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Act
+
+        ABICallElection defaultFederationElection = federationStorageProvider.getFederationElection(authorizer);
+        byte[] defaultFederationElectionSerialized = BridgeSerializationUtils.serializeElection(defaultFederationElection);
+        ABICallSpec abiCallSpec = new ABICallSpec("function1", new byte[][]{});
+        RskAddress voterAddress = new RskAddress("9be6f6735c4d59c10240d4987414fb686c6b7323");
+        defaultFederationElection.vote(abiCallSpec, voterAddress);
+        byte[] federationElectionSerializedWithVote = BridgeSerializationUtils.serializeElection(defaultFederationElection);
+
+        federationStorageProvider.save(networkParameters, activations);
+
+        // Assert
+
+        ABICallElection actualAbiCallElection = storageAccessor.getFromRepository(FEDERATION_ELECTION_KEY.getKey(), data -> BridgeSerializationUtils.deserializeElection(data, authorizer));
+        byte[] actualAbiCallElectionSerialized = BridgeSerializationUtils.serializeElection(actualAbiCallElection);
+
+        assertFalse(Arrays.equals(defaultFederationElectionSerialized, federationElectionSerializedWithVote));
```

Comment:
maybe add a comment like "check that the election with and without the vote do not match", wdyt?

Reply:
I don't think that's necessary. That's what this line is doing. We could add a comment explaining why we are asserting they should not be the same, but I think it is still unnecessary, since the code clear shows it. If you add a vote to the election, that election should be different than the election that didn't have that vote.